.gitlab/tidy-dependabot-pr.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# NOTE: this script is meant to be run on the GitLab CI, it depends on GitLab CI variables
+# Based on https://github.com/signalfx/splunk-otel-java/blob/c9134906c84e9a32a974dec4b380453fe1757410/scripts/propagate-version.sh
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck source-path=SCRIPTDIR
+source "${SCRIPT_DIR}/common.sh"
+
+ROOT_DIR="${SCRIPT_DIR}/../"
+cd "${ROOT_DIR}"
+
+tidy_dependabot_pr() {
+  local repo="signalfx/splunk-otel-collector"
+  local repo_url="https://srv-gh-o11y-gdi:${GITHUB_TOKEN}@github.com/${repo}.git"
```

Comment:
Does the automation work in gitlab if we're pointing at github here?

Reply:
We clone the github repo in the steps below and commit the changes from there instead of gitlab.